sublime_jedi/go_to.py

```diff
@@ -43,11 +44,55 @@ def _jump_to_in_window(self, filename, line_number=None, column_number=None, tra
                 self.view.show(self.point)
                 return
             filename, line_number, column_number = self.options[filename]
+
+        flags = self.prepare_layout(active_window, transient, filename)
+        active_window.open_file('%s:%s:%s' % (filename, line_number or 0,
+                                column_number or 0), flags)
+
+    def prepare_layout(self, window, transient, filename):
```

Comment:
looks like preparation and switching methods could be moved out of class.
what you think about this?